content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  Richard Belleville (Google),
+  Frank Budinsky (IBM),
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Mattia Lavacca (Kong),
+  Christine Kim (Isovalent),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other reviewer and release note contributors.
+---
+
+![Gateway API Logo](gateway-api-logo.svg)
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release of
+[Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentReference Port**,
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. Clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
```

Comment:
Maybe we should just highlight the biggest new features at this point and include more details below?
```suggestion
[Gateway API](https://gateway-api.sigs.k8s.io/). In this release, several features
are graduating to Standard Channel (GA), notably including support for Service Mesh,
and GRPCRoute. We're also introducing some new experimental features, including
Session Persistence and Client Certificate Verification. Read on for the full list of 
what's included in this release of Gateway API.
```

Reply:
Nit: should "service mesh" be lowercase in body copy?

Reply:
All suggestions added.